src/main.cpp

```diff
@@ -117,6 +117,14 @@ void setup() {
 
 void loop() {
   static long lastFlush = 0;
+  uint32_t second = 0;
+  uint32_t minute = 0;
+  uint32_t hour = 0;
+
+  hal->getLocalTime(&hour, &minute, &second);
+  if ((hour == 0) && (minute == 0) && (second == 0)) {
```

Comment:
the watch will be in deep sleep most of the time, especially at night.

I like the idea, but it won't work that easy.

We have to look into waking up the system via an RTC interrupt and check on demand to reset the step count.

Reply:
Yes you're right. For testing purpose I disabled auto sleep, so I didn't had that problem.
I'll look at it